### ISSUE TITLE
create a tool for step between xmsannotator and retip prediction

### ISSUE THREAD

**Issue by martenson** [RECETOX/galaxytools#28]
We need a tool that takes compound database and annotation as inputs and creates a list of smiles for retip prediction. 

Red oval on the screenshot below.
![LCMS_pipeline_pilot_usecase__UMSA____Microsoft_Teams](https://user-images.githubusercontent.com/1814954/96447539-53d34c80-1212-11eb-8065-7cfe70076468.png)

**Comment by martenson** [comment]
This should be possible with the Query Tool https://github.com/RECETOX/galaxytools/tree/master/tools/query

**Comment by martenson** [comment]
@ljocha Please check out the [query](https://umsa.cerit-sc.cz/root?tool_id=testtoolshed.g2.bx.psu.edu/repos/recetox/query/query/0.1) tool on umsa. It'd be nice to know whether we can use it to bridge the output from the xmsannotator to retip (and thus connect to our testing workflow at https://github.com/RECETOX/workflow-testing/tree/main/LC_MS_pipeline)

**Comment by martenson** [comment]
Is this still needed/wanted @hechth ?

**Comment by hechth** [comment]
Yes. xMSannotator outputs a table which has chemical formula and compound ID, where the ID maps back into the supplied compound database AFAIK. It should be possible to retrieve the smiles for this ID from the compound database, as it should contain the SMILES anyway.
